spikeinterface/sorters/launcher.py

```diff
@@ -169,10 +167,10 @@ def run_sorters(sorter_list,
             if need_dump:
                 if not recording.is_dumpable:
                     raise Exception('recording not dumpable call recording.save() before')
-                recording = recording.to_dict()
+                recording_arg = recording.to_dict()
             else:
-                recording = recording
-            task_args = (sorter_name, recording, output_folder, verbose, params)
+                recording_arg = recording
```

Comment:
I prefred the orevious logic with naming.

Reply:
It doesn't work if multiple jobs are used because the recording becomes a dict and at the next step it's not a recording anymore, so it throws an error

Reply:
ok